test/trento/application/integration/checks/checks_test.exs

```diff
@@ -1,17 +1,17 @@
-defmodule Trento.Integration.MapperTest do
+defmodule Trento.Integration.ChecksTest do
   @moduledoc false
 
   use ExUnit.Case
 
-  alias Trento.Messaging.Mapper
+  import Mox
 
   alias Trento.Checks.V1.{
     ExecutionRequested,
     Target
   }
 
   describe "event mapper" do
-    test "should map to a ExecutionRequested event" do
+    test "should publish an ExecutionRequested event" do
       execution_id = UUID.uuid4()
       group_id = UUID.uuid4()
       provider = :some_provider
```

Comment:
Not a biggie at all, but gh started complaining on unmodified code (`provider` and `hosts` being unused variables).
Up to you whether to make it happy.

Reply:
Thanks, we should enforce  "warning as errors" in the CI in the future. We still haven't done it because Elixir v1.14 changed something that causes the compiler to complain in certain files and I still need to understand how to make it happy.